clang/include/clang/CIR/Dialect/IR/CIROpenCLAttrs.td

```diff
@@ -147,4 +147,25 @@ def OpenCLKernelArgMetadataAttr
   let genVerifyDecl = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// OpenCLVersionAttr
+//===----------------------------------------------------------------------===//
+
+def OpenCLVersionAttr : CIR_Attr<"OpenCLVersion", "cl.version"> {
+  let summary = "OpenCL version";
+  let parameters = (ins "int32_t":$major, "int32_t":$minor);
+  let description = [{
+    Represents the version of OpenCL.
+
+    Example:
+    ```
+    // Module compiled from OpenCL 1.2.
+    module attributes {cir.cl.version = cir.cl.version<1, 2>} {}
```

Comment:
Can you please add an alias so that the attribute could be a little less verbose?

Reply:
The description updated.

Reply:
It actually looks even more verbose with the alias! Hopefully we can have a better output once we implement our owne cir module. Anyways, please go back to the previous version (without the alias), I'll approve now and merge later when you update again, sorry the back-n-forth